.travis.yml

```diff
@@ -1,18 +1,41 @@
 sudo: required
 dist: trusty
-language: generic
+language: python
 services:
   - docker
 
-cache:
-  directories:
-    - ~/.cache/pip
+
+env:
+  global:
+    - ROCKSDB_COMMIT=479c566771d6464785f205a368701c689e094fe5
+    - TWINE_USERNAME=bauerj
```

Comment:
I suppose this should be changed to an account the maintainers control?

Reply:
Yes, to quote myself:

> In order to automatically upload new releases to PyPi, you should make some changes before merging:

> - Set `TWINE_USERNAME` in the `.travis.yml` to a username that has access to python-rocksdb on PyPi.
- Using the Travis-CI Web Interface, add a hidden `TWINE_PASSWORD` variable for that user: https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings

Reply:
oops, was only reading the code :)

Reply:
btw it's probably preferable to set username to `__token__` and generate an API key specifically for travis on PyPI